src/SWP/Bundle/BridgeBundle/Resources/config/serializer/Model.BaseContent.yml

```diff
@@ -63,3 +63,7 @@ SWP\Component\Bridge\Model\BaseContent:
         keywords:
             expose: true
             type: array
+        pubStatus:
```

Comment:
I keep here the same convention as it is in the ninjs (ninjs validator) format according to IPTC and the way Superdesk handles this.